### ISSUE TITLE
Not able to run the interactsh-server

### ISSUE THREAD

**Issue by HritikHS** [projectdiscovery/interactsh#6]
When I run the command interactsh-server -domain mydomain.com -hostmaster admin@mydomain.com -ip [VPS IP]. The following output is given without any errors:
```
2021/05/01 08:45:17 Creating new order for domains: [*.mydomain.com mydomain.com]
2021/05/01 08:45:17 Order created: https://acme-v02.api.letsencrypt.org/acme/order/121967319/9409571122
2021/05/01 08:45:17 Fetching authorization: https://acme-v02.api.letsencrypt.org/acme/authz-v3/12762297545
2021/05/01 08:45:17 Fetched authorization: mydomain.com
2021/05/01 08:45:27 Updating challenge for authorization mydomain.com: https://acme-v02.api.letsencrypt.org/acme/chall-v3/12762297545/FkqiUg
```

But the server is not giving A records when queried. I already had a letsencrypt cert but even after I deleted the cert nothing changed the same output is given

**Comment by Ice3man543** [comment]
@HritikHS can you provide more details including your environment and steps you took for installing and running the server?

**Comment by HritikHS** [comment]
Yeah sure, I am using Ubuntu 20.04.1 LTS (GNU/Linux 5.4.0-42-generic x86_64) and I followed the steps mentioned at [https://github.com/projectdiscovery/interactsh#setting-up-self-hosted-instance](https://github.com/projectdiscovery/interactsh#setting-up-self-hosted-instance)

**Comment by ehsandeep** [comment]
@HritikHS what domain provider you are using for this? for steps 2 and 3?

**Comment by Ice3man543** [comment]
@HritikHS also can you run the server with -debug. That will give more context on error and paste the output here.

**Comment by Ice3man543** [comment]
I think I got the problem, you need to use debian since ubuntu already has a dns server running locally. Last time we tried it was very messy to remove that dns server. So you should try on Debian

**Comment by HritikHS** [comment]
@Ice3man543 you were right it was due to the conflict with systemd-resolved dns server running on 0.0.0.0:53 but this can be resolved by using the VPS IP instead of 0.0.0.0 to run the DNS server. I have created a pull request for that and thank you so much @ehsandeep @Ice3man543 for all the wonderful tools from Project Discovery.

**Comment by Ice3man543** [comment]
Closing the issue as merged PR #11 fixes this.
